packages/strapi-design-system/src/Combobox/Combobox.tsx

```diff
@@ -60,6 +60,8 @@ export interface ComboboxInputProps
   placeholder?: string;
   size?: 'S' | 'M';
   startIcon?: React.ReactNode;
+  'aria-label'?: string;
+  'aria-describedby'?: string;
```

Comment:
are these not carried through via the `Omit<ComboboxPrimitive.TextInputProps>`?

Reply:
I missed it